### PR TITLE
Add community roadmap update

### PR DESCRIPTION
**Pull request by melcorr**
https://melcorr.github.io/pulpproject.org/2021/08/24/pulp-community-roadmap-update/

@RCMariko feel free to tag anyone who you think should be reviewing this!

### CODE REVIEW

**Comment by melcorr** [comment]
@RCMariko you mentioned that you wanted a few folks to review this. 
Can you add who you think should look at it?

**Review by RCMariko — APPROVED** [review]

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (goosemania) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: I'd leave it to @dkliban , however I suggest to mention examples when OSTree plugin is useful, why we decided to add it back, or do not mention Atomic Host and its EOL.
It's a bit confusing to me, I read it as OSTree plugin has nothing to manage, but hurray we are adding it :D

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (daviddavis) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: Is this doc meant to cover 3.15? Several of these items (Encrypted fields, Disk usage, Orphan cleanup, Django 3 and Python 3.7+ support) went out with 3.15 already.

Comment (daviddavis) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: Maybe worth adding here that we are removing the old tasking system and hardening the new one? cc @mdellweg

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (daviddavis) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]:
```suggestion
* ACS support
```

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (melcorr) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: This was done a while ago and was in a google doc waiting for review, then got promoted to a PR waiting for review.

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (daviddavis) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: :+1:

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (daviddavis) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: Small suggestion: link to our new discourse.

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (bmbouter) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]:
```suggestion
* Alternate Content Source support
```

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (bmbouter) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: Can we elaborate a bit on this maybe? I assume it's to reduce disk usage.

Comment (bmbouter) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: Maybe `Reduce Disk Usage while Syncing` ?

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (RCMariko) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: My bad that was my suggestion on mentioning the going away stuff. @dkliban can perhaps provide some suggested wording to focus on current/future.

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (bmbouter) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]:
```suggestion
* Encrypting sensitive fields inside the database
```

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (mdellweg) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]:
```suggestion
Further updates will be announced on the Pulp blog and community. If you have any questions or feedback, feel free to write or post to our community channels!
```

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (mdellweg) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: Isn't this done? Or, what is left?

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (mdellweg) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: This is also released with 3.15

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (mdellweg) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: "* Removing the legacy tasking system in favor of the distributed one introduced in 3.14."

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (mdellweg) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]:
```suggestion
* API token support
```
It's going to share the token with pulp_ansible. So maybe call it the same way there.

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (mdellweg) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: finished?

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (mdellweg) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: This is really unclear to me. Is it about the ability to install all dot-Releases from the same minor branch with the same installer version?

Comment (melcorr) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: No clue. I will look into it ;)

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (melcorr) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: This was also in 3.15 now that I look at it, wasn't it ..

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (melcorr) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: Probably done. This was written a while before the 3.15 release.

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (melcorr) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: Yes. You're all too quick

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (RCMariko) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: @mikedep333 can you clarify our roadmap for installer plans?

Comment (mikedep333) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]:
```suggestion
* pulp_installer 3.16.z will support installing any pulpcore 3.16.z release
```

Thoughts @dkliban ?

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (melcorr) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: @daviddavis is there anything I could add here for all of the items I'm removing

**Comment by melcorr** [comment]
If anyone has anything we should *add* since I removed a lot of stuff, that would be great. 
Only thing I haven't really dealt with is the feedback on OSTree plugin. 
If I hear nothing back, I'll look through some tickets and try to invent something.

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (daviddavis) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: Not that I'm aware of. @pulp/core anyone know of anything?

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (daviddavis) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]:
```suggestion
* Alternate Content Sources - [check out the epic](https://pulp.plan.io/issues/7832) to learn more about what this involves
```

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (ipanova) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: s/Alternative/Alternate/

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (ipanova) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: is it worth mentioning any in progress or in plan articles?

Comment (ipanova) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: should we also mention in any way community survey feedback audit/AIs?

Comment (melcorr) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: :-) I suppose if we need some filler.

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (ipanova) [inline _posts/2021-08-24-pulp-community-roadmap-update.md]: DRF token support during registry login and sync https://pulp.plan.io/issues/9115

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (ipanova) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: What about mentioning Content Signing and evaluation of integration with Sigstore?

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (goosemania) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: @ipanova , I wouldn't call out specifically the DRF token. There are security concerns about it, and it's not decided which library will be used for the token support. And also this work got postponed a bit, we are not under the gun to deliver it now, so the progress is slow in this area and many things are unclear. That's my understanding of the situation.

Comment (ipanova) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: ok, thanks for the updates. There is already token support in this plugin that's why there is the need to distinguish the type to not create additional confusion.

Comment (melcorr) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: I can just remove this if it is confusing.

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (melcorr) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: Thanks!

File: _posts/2021-08-24-pulp-community-roadmap-update.md
Comment (ipanova) [reply _posts/2021-08-24-pulp-community-roadmap-update.md]: we can just leave a generalized note 'Token support enhancements'

**Review by ipanova — APPROVED** [review]

**Comment by melcorr** [comment]
If it's good enough for Ina, it's good enough for me.  Will leave a little bit just to be sure, then update the date and merge.

**Review by daviddavis — APPROVED** [review]

**Comment by melcorr** [comment]
Thank you all!

**Pull request merged**
